buildSrc/src/main/groovy/fahrschein.java-conventions.gradle

```diff
@@ -49,6 +55,7 @@ configurations.all {
     resolutionStrategy.dependencySubstitution {
         substitute(platform(module('commons-logging:commons-logging'))).
                 using module("org.slf4j:jcl-over-slf4j:${property('slf4j.version')}")
+        // hamcrest changed package coordinates. Make sure to only include the new one.
         substitute(platform(module('org.hamcrest:hamcrest-core'))).
                 using module("org.hamcrest:hamcrest:2.2")
```

Comment:
Why is there the need for using hamcrest at all? I'm a huge fan of defaults and if there is no specific use case I would be in favor of dropping it.

Reply:
I agree that the test code can be written without hamcrest, but it's a actively maintained and well-known library, and removing would mean another major refactoring. E.g. MockServer is using hamcrest matchers as part of its verification interface. 

We should rather look at our use of org.hobsoft.hamcrest:hamcrest-compose to see if that's really needed. I guess  that can be replaced with junit5's 'assertAll' ...